api/migrations/2020-11-08-174607_trigger_users/down.sql

```diff
@@ -0,0 +1,2 @@
+DROP TRIGGER IF EXISTS delete_dangling_records;
+DROP FUNCTION IF EXISTS delete_dangling_program_records;
```

Comment:
I think we shouldn't have the `IF EXISTS` here, cause if this DROP fails that means something about it is broken, so we wanna know. Like hypothetically if you misspelled a function name here we wouldn't know.

Also I don't think the test suite actually runs these `down` migrations, we should prob make that happen at some point.